commands.json

```
@@ -360,14 +360,18 @@
       "description": "Start the GraphQL API server with caching and limits",
       "cmd": [
         "squid-graphql-server",
+        "--max-root-fields",
+        "30",
+        "--max-response-size",
+        "5000",
```

```suggestion
        "--max-root-fields",
        "30",
        "--max-response-size",
        "5000",
        "--max-request-size",
        "1000",
        "--subscription",
        "false"
```

Can maybe add these two? What do you think about disabling subscriptions?

---

I like this. We should use subscriptions for the UI at some point though. Think it may be useful.